src/scenarios/netcoreapp/test.py

```diff
@@ -0,0 +1,21 @@
+import os
+from shared.runner import TestTraits, Runner
+from shared import const
+
+SCENARIO_NAME = '.NET Core 2.0 Console Template'
+EXE_NAME = 'NetCoreApp'
+
+def main():
+    traits = TestTraits(scenarioname=SCENARIO_NAME,
+                        exename=EXE_NAME,
+                        guiapp='false',  # string passed through to tool
+                        sdk=True,
+                        startup=True,
```

Comment:
This should not be a startup test since we never run a `netcoreapp2.0` compatible framework.

Reply:
do you mean startup=True implies running the app's executable under netcoreapp2.0 framework?

Reply:
Not precisely. `startup=True` will attempt to run the startup harness using the generated app as the target. That will fail because we have no `netcoreapp2.0` runtime on the test machine (unless we are running `netcoreapp2.0` tests, which we don't measure.)